templates/song_rater/song_detail.html

```diff
@@ -27,5 +27,6 @@ <h2>Rate The Song</h2>
        <td>Sorry, no ratings in this list.</td>
        {% endfor %}
     </ul>
+   <li><a href="{% url 'song_rater:index'%}">Back to homepage</a></li>
```

Comment:
Nice that you added a link here as well.  Same remark on the `li` element as above

Reply:
Also, A "back to songs list" link might make more sense.